pkg/system/phase1_verifying.go

```diff
@@ -252,3 +273,26 @@ func (r *Reconciler) checkExternalPg(postgresDbURL string) error {
 	}
 	return nil
 }
+
+func (r *Reconciler) checkBucketLoggingPVC() error {
+	BucketLoggingPVC := &corev1.PersistentVolumeClaim{
+		TypeMeta: metav1.TypeMeta{Kind: "PersistenVolumeClaim"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      *r.NooBaa.Spec.BucketLogging.BucketLoggingPVC,
+			Namespace: r.Request.Namespace,
+		},
+	}
+	if !util.KubeCheck(BucketLoggingPVC) {
+		return util.NewPersistentError("InvalidBucketLoggingConfiguration",
+			"'Guaranteed' BucketLogging requires a Persistent Volume Claim (PVC) with ReadWriteMany (RWX) access mode. The specified 'BucketLoggingPVC' was not found")
```

Comment:
I think we can take out all of beginning part: 
```suggestion
			"The specified 'BucketLoggingPVC' was not found")
```